main_test.go

```diff
@@ -383,6 +383,53 @@ func TestPublishersEndpoints(t *testing.T) {
 				assert.Equal(t, "wrong cursor format in page[after] or page[before]", response["detail"])
 			},
 		},
+		{
+			description: `GET with "from" query param`,
+			query:       "GET /v1/publishers?from=2018-11-26T00:56:23Z",
+
+			expectedCode:        200,
+			expectedContentType: "application/json",
+			validateFunc: func(t *testing.T, response map[string]interface{}) {
+				assert.IsType(t, []interface{}{}, response["data"])
+				data := response["data"].([]interface{})
+
+				assert.Equal(t, 13, len(data))
```

Comment:
```suggestion
				assert.Equal(t, 14, len(data))
```

...so this assertion is different than the `to=` one. Having the same number can make one think `13` is special for some reason and use it in future tests, reducing variance.

(just to be extra-cautious)